fabric-common/types/index.d.ts

```diff
@@ -257,6 +257,11 @@ export class EventService extends ServiceAction {
 	isStarted(): boolean;
 }
 
+export interface StartEventRequest {
+	targets: Eventer[];
```

Comment:
I think this is actually an optional property:
```
targets?: Eventer[];
```
the JSDoc would need to be updated to show it as optional too.